docs/en/writing-running-appium/server-args.md

```diff
@@ -40,20 +40,20 @@ All flags are optional, but some are required in conjunction with certain others
 |`--nodeconfig`|null|Configuration JSON file to register appium with selenium grid|`--nodeconfig /abs/path/to/nodeconfig.json`|
 |`-ra`, `--robot-address`|0.0.0.0|IP Address of robot|`--robot-address 0.0.0.0`|
 |`-rp`, `--robot-port`|-1|port for robot|`--robot-port 4242`|
-|`--chromedriver-port`|9515|Port upon which ChromeDriver will run|`--chromedriver-port 9515`|
-|`--chromedriver-executable`|null|ChromeDriver executable full path||
+|`--driver-args '{"uiautomator2": {"chromedriver-port": 9515}}`|9515|Port upon which ChromeDriver will run|`--driver-args '{"uiautomator2": {"chromedriver-port": 9515}}`|
+|`--driver-args '{"uiautomator2": {"chromedriver-executable": '/path/to/file'}}`|null|ChromeDriver executable full path||
```

Comment:
since we're doing JSON, we should use camelCase for all arg names, so `chromedriverExecutable`

Reply:
if these capabilities are the same as driver capabilities then it makes sense to just provide default capabilities json instead to simplify their validation. Also, I would prefer to have a possibility to load that stuff either from JSON or YAML file, which might be more convenient taking into account the amount of formatting that should be applied to the command line